kind/common.sh

```diff
@@ -65,11 +65,17 @@ aarch64* | arm64*)
     ;;
 esac
 
+# check os
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
I know sed on mac has different behaviors with linux. hence if the change is not impact with linux(for our CI), it looks go to me.

Reply:
Yes it won't affect other Linux distributions